test/Microsoft.ML.Tests/Transformers/HashTests.cs

```diff
@@ -261,9 +278,20 @@ private void HashTestPositiveIntegerCore(ulong value, uint expected, uint expect
         [Fact]
         public void TestHashIntegerNumbers()
         {
+            //32bit
+            //HashTestPositiveInteger32BitCore(0, 842, 358, 20);
+            //HashTestPositiveInteger32BitCore(1, 502, 537, 746);
+            //HashTestPositiveInteger32BitCore(2, 407, 801, 652);
+
```

Comment:
Please remove the commented lines here if they are not meant to be part of the code.